ms-engine/src/main/java/net/trajano/ms/engine/internal/VertxInputStream.java

```diff
@@ -0,0 +1,185 @@
+package net.trajano.ms.engine.internal;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.concurrent.Semaphore;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.streams.ReadStream;
+
+/**
+ * <p>
+ * </p>
+ * An input stream that wraps a Vert.X Read Buffer. This does not do anything
+ * that performs blocks.
+ * </p>
+ * <p>
+ * This pauses() the stream once the buffer is read and resumes the stream when
+ * the buffer is fully read.
+ * </p>
+ */
+public class VertxInputStream extends InputStream {
+
+    private static final Logger LOG = LoggerFactory.getLogger(VertxInputStream.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)